source/SyntheticTopology/Overtness.lagda

```diff
@@ -0,0 +1,101 @@
+---
+title:        Overtness in Synthetic Topology
+author:       Martin Trucchi
+date-started: 2024-05-28
+---
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import MLTT.Spartan
+open import SyntheticTopology.SierpinskiObject 
+open import UF.Base
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Sets
+open import UF.Sets-Properties
+open import UF.Subsingletons
+open import UF.SubtypeClassifier
+open import UF.UniverseEmbedding
+
+module SyntheticTopology.Overtness
+        (𝓤  𝓥 : Universe)
+        (fe : Fun-Ext)
+        (pe : Prop-Ext)
+        (pt : propositional-truncations-exist)
+        (𝕊 : Generalized-Sierpinski-Object fe pe pt 𝓤 𝓥)
+        where
+
+open import UF.ImageAndSurjection pt
+open import UF.Logic
+
+open AllCombinators pt fe
+open PropositionalTruncation pt hiding (_∨_)
+open Sierpinski-notations fe pe pt 𝕊
+
+
+\end{code}
+
+Overtness
+
+This notion is similar to compactness, in a dual meaning.
+A set is overt if the proposition (∃ x , P x) is open whenever P is open.
+
+\begin{code}
+
+is-overt : hSet 𝓤  → Ω (𝓤 ⁺ ⊔ 𝓥)
+is-overt (X , sX) =
+  Ɐ (P , open-P) ꞉ 𝓞 (X , sX) ,
+   is-open-proposition (Ǝₚ x ꞉ X , P x)
+
+\end{code}
+
+We prove here, similar to image-of-compact, that image of overt sets are overt. 
```

Comment:
When you refer to a piece of code in a comment, can you wrap it in backticks so that Markdown compilers render it as a code block?

Reply:
Done!